src/pages/careers/_model/_locations/_contact-details.js

```diff
@@ -1,25 +1,43 @@
 import { localize } from 'components/localization'
 
-const address_cyberjaya = localize(
-    'C-13, iTech Tower, Jalan Impact, Cyber 6, 63000 Cyberjaya, Selangor, Malaysia.',
-)
-const address_dubai = localize('1902, Jumeirah Business Center 1, JLT Cluster G, Dubai, UAE.')
+const address_cyberjaya = [
+    localize('iTech Tower, Jalan Impact, Cyber 6,'),
```

Comment:
career page doesnt need localize, applied to all other array localize

Reply:
Done 👌